script/tool/lib/src/drive_examples_command.dart

```diff
@@ -123,6 +123,11 @@ class DriveExamplesCommand extends PackageLoopingCommand {
 
   @override
   Future<PackageResult> runForPackage(RepositoryPackage package) async {
+    if (!package.requiresFlutter()) {
+      return PackageResult.skip('Not a Flutter package.');
+    }
```

Comment:
Should this check be performed in the identified examples below?

We have an incoming PR where somebody is [removing the `flutter` dependency for the `cross_file` package](https://github.com/flutter/packages/pull/1664) (which does have merit), will we need a special run_tests.sh there to run the tests?

(PS: `cross_file` doesn't have an `example` with tests yet, but I guess a simple one would be a `flutter` app that uses cross_file for something)

Reply:
This is a really good point. I had that check because the repo tooling has always only returned Flutter packages, not all packages, when iterating over examples. But there's no reason that should be true, so I fixed that and just removed the check entirely. (We don't need the check at the example level because nobody should ever try to set up Flutter driver for a non-Flutter example; if they do, it's on them when CI fails 🙂 )

This fix will save us some confusion when someone eventually tries to use the new excerpt support in a non-Flutter package in flutter/packages, because that is based on example iteration too.